doc/rtd/reference/yaml_examples/update_upgrade.rst

```diff
@@ -0,0 +1,35 @@
+.. _cce-update-upgrade:
+
+Update, upgrade and install packages
+************************************
+
+These examples show you how to install, update, and upgrade packages during
+boot. By default, this is set to "none", but if any packages are specified
```

Comment:
This isn't true. If `upgrade` is specified, `update` will automatically run, but the presence of packages doesn't invoke an update.

Reply:
Thanks, removed